tests/components/homekit/test_util.py

```diff
@@ -199,27 +204,36 @@ async def test_show_setup_msg(hass):
     """Test show setup message as persistence notification."""
     pincode = b"123-45-678"
 
-    call_create_notification = async_mock_service(hass, DOMAIN, "create")
+    entry = await async_init_integration(hass)
+    assert entry
+
+    call_create_notification = async_mock_service(
+        hass, PERSISTENT_NOTIFICATION_DOMAIN, "create"
+    )
 
-    await hass.async_add_executor_job(show_setup_message, hass, pincode, "X-HM://0")
+    await hass.async_add_executor_job(
+        show_setup_message, hass, entry.entry_id, "bridge_name", pincode, "X-HM://0"
+    )
     await hass.async_block_till_done()
-    assert hass.data[HOMEKIT_PAIRING_QR_SECRET]
-    assert hass.data[HOMEKIT_PAIRING_QR]
+    assert hass.data[DOMAIN][entry.entry_id][HOMEKIT_PAIRING_QR_SECRET]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5453535)